ir/value.h

```diff
@@ -83,4 +83,12 @@ class Input final : public Value {
   smt::expr getTyVar() const;
 };
 
+class VectorValue final : public Value {
```

Comment:
Is this class still needed?  I added a ConstantAggregate class that should be sufficient for vectors, structs and arrays.

Reply:
Good point, I've removed this function.